objects/rct2/scenery_group/rct2.scenery_group.scgabstr.json

```diff
@@ -14,12 +14,15 @@
             "rct2.scenery_small.tge5",
             "rct2.scenery_small.tgc1",
             "rct2.scenery_small.tgc2",
+            "rct2.scenery_wall.wgw2",
             "rct2.scenery_wall.wallgl8",
             "rct2.scenery_wall.wallgl16",
             "rct2.scenery_wall.wallgl32",
             "rct2.scenery_small.georoof1",
-            "rct2.scenery_wall.wgw2",
-            "rct2.scenery_small.georoof2"
+			"toontowner.scenery_small.ttrfgl01",
```

Comment:
Make sure you use spaces instead of tabs next time. I will fix it for this PR.

Reply:
Oh that's odd, i typed in spaces but copied it over, guess notepad++ converted them to tabs. I'll keep it in mind.